EIPS/eip-7781.md

```diff
@@ -0,0 +1,186 @@
+---
+eip: 7781
```

Comment:
```suggestion
eip: 7782
```
Assigning the EIP/ERC/RIP number.  To prevent any appearance of number sniping, I am bumping the number.  EIP numbers are manually assigned by editors and associates (there is a tracking spreadsheet that we maintain).

Please also update the filename.